src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/electric/machines/FluidPump.java

```diff
@@ -130,12 +130,12 @@ protected void tick(Block b) {
     }
 
     private void consumeFluid(Block fluid) {
-        if (fluid.getType() == Material.WATER) {
+        if (fluid.getType() == Material.STATIONARY_WATER) {
```

Comment:
I dont think STATIONARY_WATER exists anymore.

Reply:
> I dont think STATIONARY_WATER exists anymore.

Oops fixed. I thought it did lol.